internal/util/connection.go

```diff
@@ -62,6 +62,11 @@ func (cc *ClusterConnection) Connect(monitors string, cr *Credentials) error {
 	return nil
 }
 
+// getRadosConnection returns the rados connection of the cluster.
```

Comment:
It is very ugly to expose the internal RadosConnection to callers. Better do something like `GetFSAdmin()` below.

Reply:
Yes this is much better :+1: